### ISSUE TITLE
test_char is failing on Fedora

### ISSUE THREAD

**Issue by igor-raits** [Goncalerta/proc-quote#19]
```rust
---- test_char stdout ----
thread 'test_char' panicked at 'assertion failed: `(left == right)`
  left: `"\'\\u{0}\' \'#\' \'\"\' \'\\\'\' \'\\n\' \'\\u{2764}\'"`,
 right: `"\'\\u{0}\' \'#\' \'\"\' \'\\\'\' \'\\n\' \'❤\'"`', tests/types.rs:70:5
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

Not sure what other info you need.

**Comment by Goncalerta** [comment]
I'm not sure what is causing this.

Does the same thing happen with the [`quote`](https://github.com/dtolnay/quote) crate?

**Comment by igor-raits** [comment]
No, the tests from `quote` crate do not fail for us in the same environment.

**Comment by Goncalerta** [comment]
That is weird. The test interpolates `#heart`; on the other hand, `interpolate_to_tokens_ident` just calls `append_to_tokens`, which calls the same trait as the original crate. 

I'm gonna temporarily remove the heart test, so it doesn't block you until the problem is understood.

**Comment by igor-raits** [comment]
No need to disable it, I am skipping it when packaging.

**Comment by Goncalerta** [comment]
It was also making travis mysteriously fail since last commit (although it still works fine on my machine and the only change in that commit was the new license files, which have no impact on this). So it is indeed a very strange behavior.
